samples/boards/stm32/pm_lptim/sample.yaml

```diff
@@ -0,0 +1,15 @@
+sample:
+  name: Power Management LP Timer
+tests:
+  sample.boards.stm32.pm_lptim.lsi:
+    build_only: true
+    depends_on: lptim
+    filter: dt_compat_enabled_with_alias("gpio-leds", "led0") and
+            dt_compat_enabled("st,stm32-lptim")
+  sample.boards.stm32.pm_lptim.lse:
+    build_only: true
```

Comment:
Should we consider changing the test os that it could be not only build but also executed. and thus be run on test bench ?

Reply:
@FRASTM ?

Reply:
When executing the test case automatically, it happens that console is no more responding
That's what the https://github.com/zephyrproject-rtos/zephyr/issues/31284 reports.
Fixing this is prefered before adding the test case to the bench.